api/v1alpha1/podhttpchaos_types.go

```diff
@@ -92,6 +96,15 @@ type PodHttpChaosSelector struct {
 	ResponseHeaders map[string]string `json:"response_headers,omitempty"`
 }
 
+type PodHttpChaosReplaceBodyAction []byte
+
+func (p PodHttpChaosReplaceBodyAction) MarshalJSON() ([]byte, error) {
+	return json.Marshal(PodHttpChaosPatchBodyAction{
```

Comment:
The name of `PodHttpChaosReplaceBodyAction` and  `PodHttpChaosPatchBodyAction` looks similar, but they represents different struct. And we have another struct `PodHttpChaosReplaceActions`,  we may be confused by their names and not know the relationship between them. Can we rename it? For example, `PodHttpChaosPatchBodyActionData`?

Reply:
Although they represents different struct, they have the same function. Both `PodHttpChaosReplaceBodyAction` and `PodHttpChaosPatchBodyAction` will be marshalled to json string like {"type":"XXX", "content":"aaaaa"}.